cmd/key-transparency-server/frontend.go

```diff
@@ -40,7 +41,7 @@ import (
 	"github.com/golang/protobuf/jsonpb"
 	"github.com/golang/protobuf/proto"
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
-	"golang.org/x/net/context"
+	oldcontext "golang.org/x/net/context"
```

Comment:
Hopefully gRPC will close issue grpc/grpc-go#711 and we won't have to use two context objects.